README.md

```diff
@@ -1,322 +1,329 @@
-> **Note**
+> **WARNING**
 > This driver is in beta release and not recommended for production use. It operates with the Fauna database service via an API which is also in beta release, and is not recommended for production use. This driver is not compatible with v4 or earlier versions of Fauna. If you would like to participate in the private beta program please contact product@fauna.com.
 
-## Table of Contents
+# A JavaScript driver for [Fauna](https://fauna.com).
 
-1. [Quick Start](#quick-start-for-the-fql-x-driver)
-1. [Learning how to use the Driver](#learning-how-to-use-the-fql-x-driver)
-1. [Understanding the wire protocol](#understanding-the-fql-x-wire-protocol)
-1. [Development](#development)
+[![Npm Version](https://img.shields.io/npm/v/fauna.svg?maxAge=21600)](https://www.npmjs.com/package/fauna)
+[![License](https://img.shields.io/badge/license-MPL_2.0-blue.svg?maxAge=2592000)](https://raw.githubusercontent.com/fauna/fauna-js/main/LICENSE)
 
-# Usage
-
-The FQL X JS driver uses a template-based approach to composing queries and operations. The advantage of this design is that you can prototype and test your queries in the Fauna dashboard shell, and then cut-and-paste those queries as templates in your client, which are executed in Fauna via this driver. You can parameterize your query by adding placeholders to the template, and then pass a set of arguments to the query() method, or resolve the placeholders with string interpolation.
-
-This driver can only be used with FQL X, and is not compatible with earlier versions of FQL. This README contains detailed instructions on how to use the driver, split into the following sections .
-
-# Quick Start for the FQL X driver
-
-## Set-up and Dependencies
+See the [Fauna Documentation](https://fqlx-beta--fauna-docs.netlify.app/fqlx/beta/) for additional information how to configure and query your databases.
 
-This driver is on NPM.
+This driver can only be used with FQL X, and is not compatible with earlier versions of FQL. To query your databases with earlier API versions, see the [faunadb](https://www.npmjs.com/package/faunadb) package.
 
-You can install with npm with `npm install fauna`.
-You can install with yarn with `yarn add fauna`
+<details>
+<summary>Table of Contents</summary>
 
-## Pre-requisites
+- [A JavaScript driver for Fauna.](#a-javascript-driver-for-fauna)
+- [Compatibility](#compatibility)
+- [Installation](#installation)
+  - [Node.js](#nodejs)
+  - [Browsers](#browsers)
```

Comment:
IMO having quick starts at the tops of READMEs makes getting ramped up on a new dependency much easier.

I'd suggest leading with a quick start that includes installation and basic usage. And moving compatibility below that.